### PR TITLE
Errata

### PR DESCRIPTION
**Pull request by swopebe**
open-cluster-management/backlog#8341--errata draft

### CODE REVIEW

**Review by blkingrh — APPROVED** [review]
Only comment is 7153 looks like it listed twice. Thanks!

File: release_notes/errata.adoc [hunk 1]
```diff
@@ -5,6 +5,53 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixed routing issues in policy pages to show `No resource` instead of a loading animation when users navigate to a URL for a policy that does not exist. (GitHub 7445)
+
+* Fixed issue with policy editor crashing when content was copied, pasted, and fixed an error in `.spec.policyTemplate` when the form did not update to display that there was a custom specification. (GitHub 7380)
+
+* Added channel connection failure message can be found in subscription status. (GitHub 7177)
+
+* Excluded channels from removable application resources, which were listed in _Delete application_ modal in the console. Now channels cannot be deleted in this modal. Only just subscriptions and placement rules are removable in this modal with this fix. (GitHub 7153)
+
+* Excluded channels from removable application resources, which were listed in _Delete application_ modal in the console. Now channels cannot be deleted in this modal. Only just subscriptions and placement rules are removable in this modal with this fix. (GitHub 7153)
```
Comment (cadawson) [inline release_notes/errata.adoc]: Is "Only just" correct here?

File: release_notes/errata.adoc [hunk 2]
```diff
@@ -5,6 +5,53 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixed failure with Git channel connection to a private Git repo with missing `secretRef` leading to `multicluster-operators-hub-subscription` pod crash. (GitHub 8764)
+
+* Fixed `cert-manager-webhook` failure to start because of permission problems with {ocp-short} 4.6.10 installation. (GitHub 8517)
+
+* Fixed a high availability configuration running too many competing compactors so that with the fix, there is only one compactor running. (GitHub 7676)
+
+* Fixed a potential performance issue where some Grafana dashboards auto-refresh  with an interval that was smaller than the metrics scrape interval. (GitHub 7665)
+
+* Added support for IBM to import IBM ROKS cluster to manage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894778[Bugzilla 1894778]).
```
Comment (cadawson) [inline release_notes/errata.adoc]: Should we remove "for IBM", or was it specifically for IBM? If so, why are we telling everyone about it?

**Review by cadawson — COMMENTED** [review]
Thanks for doing these, Brandi!

File: release_notes/errata.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/errata.adoc]: nope lol

File: release_notes/errata.adoc (hunk 2 above)
Comment (swopebe) [reply release_notes/errata.adoc]: Yes, RHOKS was an integrated product that we had doc for at IBM, if you remember.... but I do need to look up the actual name and change the term here. Thanks for the reminder. I will check that as well.

Comment (swopebe) [reply release_notes/errata.adoc]: spelled out and made it clear that is was a managed IBM Cloud cluster.

File: release_notes/errata.adoc (hunk 1 above)
Comment (swopebe) [reply release_notes/errata.adoc]: removed just

File: release_notes/errata.adoc [hunk 3]
```diff
@@ -5,6 +5,50 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixed configuration policy controller crash when handling policies. (GitHub 7569)
+
+* Fixed missing namespace when editing an existing provider connection. (GitHub 7501)
+
+* Fixed routing issues in policy pages to show `No resource` instead of a loading animation when users navigate to a URL for a policy that does not exist. (GitHub 7445)
+
+* Fixed issue with policy editor crashing when content was copied, pasted, and fixed an error in `.spec.policyTemplate` when the form did not update to display that there was a custom specification. (GitHub 7380)
+
+* Added channel connection failure message can be found in subscription status. (GitHub 7177)
```
Comment (dockerymick) [inline release_notes/errata.adoc]: I suggest the following updates:

Added channel connection failure message can be found in subscription status.

TO 

Added channel connection failure message, which can be found in the subscription status.

File: release_notes/errata.adoc [hunk 4]
```diff
@@ -5,6 +5,50 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Excluded channels from removable application resources, which were listed in _Delete application_ modal in the console. Now channels cannot be deleted in this modal. Only subscriptions and placement rules are removable in this modal with this fix. (GitHub 7153)
+
+* Fixed display of NIST categories, standards, and controls for consistency across all policy elements and adjusted to NIST content. (GitHub 6954)
+
+* Increased the search pod memory requests and limits in default install to handle most workloads without intervention: Memory limit for Search `redisgraph` pod to 4G,
+memory request for Search API and Redisgraph pods to 128M. (GitHub 6890)
+
+* Fixed failure with Git channel connection to a private Git repo with missing `secretRef` leading to `multicluster-operators-hub-subscription` pod crash. (GitHub 8764)
```
Comment (dockerymick) [inline release_notes/errata.adoc]: I would spell out repo. Change to repository.

File: release_notes/errata.adoc [hunk 5]
```diff
@@ -5,6 +5,50 @@ By default, Errata updates are automatically applied when released. See link:../
...
+
+* Fixed a high availability configuration running too many competing compactors so that with the fix, there is only one compactor running. (GitHub 7676)
+
+* Fixed a potential performance issue where some Grafana dashboards auto-refresh  with an interval that was smaller than the metrics scrape interval. (GitHub 7665)
+
+* Added support to import Red Hat OpenShift on IBM Cloud cluster to manage. (link:https://bugzilla.redhat.com/show_bug.cgi?id=1894778[Bugzilla 1894778])
+
+* Fixed Git webhook notification function to deploy the selected Git repository resources to target clusters through a subscription. (GitHub 6785)
+
+* Fixed Application topology resources deployed successfully but not accessible offline. Now the cluster node displays a failed status if any remote clusters are offline. (GitHub 6298)
```
Comment (dockerymick) [inline release_notes/errata.adoc]: Change 

Fixed Application topology resources deployed successfully but not accessible offline.

TO 

Fixed Application topology resources to deploy successfully but are not accessible offline.

**Review by dockerymick — COMMENTED** [review]
Thanks Brandi for your updates!

File: release_notes/errata.adoc (hunk 3 above)
Comment (swopebe) [reply release_notes/errata.adoc]: good point, thanks!

File: release_notes/errata.adoc [hunk 6]
```diff
@@ -5,6 +5,49 @@ By default, Errata updates are automatically applied when released. See link:../
...
+* Fixed issue with policy editor crashing when content was copied, pasted, and fixed an error in `.spec.policyTemplate` when the form did not update to display that there was a custom specification. (GitHub 7380)
+
+* Added channel connection failure message, which can be found in the subscription status. (GitHub 7177)
+
+* Excluded channels from removable application resources, which were listed in _Delete application_ modal in the console. Now channels cannot be deleted in this modal. Only subscriptions and placement rules are removable in this modal with this fix. (GitHub 7153)
+
+* Fixed display of NIST categories, standards, and controls for consistency across all policy elements and adjusted to NIST content. (GitHub 6954)
+
+* Increased the search pod memory requests and limits in default install to handle most workloads without intervention: Memory limit for Search `redisgraph` pod to 4G,
+memory request for Search API and Redisgraph pods to 128M. (GitHub 6890)
```
Comment (cadawson) [inline release_notes/errata.adoc]: MB?
Now that I look at it again, maybe GB, as well? I read it as 4G (as in network) the first time.

**Review by cadawson — APPROVED** [review]
One quick question about MB, but looks good, otherwise. Thanks!

**Pull request merged**